.github/CODEOWNERS

```diff
@@ -0,0 +1,98 @@
+# Numba's codeowners file is dual purpose, it:
+#
+# 1. Provides information to github about who should be requested to review a PR
+# 2. Provides users/contributors/czars general information about who to contact
```

Comment:
I'm not sure we should encourage users to directly contact specific developers about parts of the code base, as it might lead to people bypassing issues/discourse instead of opening an issue and allowing the FPOC on duty to triage the issue appropriately - it is often not clear what part of the code base might really be the source of an issue.

```suggestion
# 2. Provides contributors/czars general information about who to contact
```

Reply:
Good point.